tests/foreman/cli/test_user.py

```diff
@@ -111,18 +140,13 @@ def test_positive_create_user_1(self, test_data):
         @Assert: User is created
 
         """
-        user = make_user(test_data)
-        self.__assert_exists(user)
+        include_list = [gen_string("alphanumeric", 100)]
+        for login in valid_data_user() + include_list:
```

Comment:
valid_data_user() will provide standard data set and include_list will have any specific data to be tested.